packages/office-ui-fabric-react/src/components/DetailsList/ShimmeredDetailsList.types.ts

```diff
@@ -29,9 +34,15 @@ export interface IShimmeredDetailsListProps extends IDetailsListProps {
 
   /**
    * Custom styles to override the styles specific to the ShimmeredDetailsList root area.
+   * @deprecated Use `styles` prop instead
```

Comment:
If `shimmerOverlayStyles` isn't used anywhere I think the deprecation comment should make that clear, like adding "Any value here will be ignored."

Reply:
Changing